### PR TITLE
tiup: add more fields and links from customized-montior-in-tiup-environment.md

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -602,6 +605,7 @@ monitoring_servers:
 - `password`：Grafana 对应的密码
 - `dashboard_dir`：该字段指定一个本地目录，该目录中应当含有完整的 `dashboard(*.json)` 文件，这些文件会在集群配置初始化阶段被传输到目标机器上，作为 Grafana 的 dashboards
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `config`：该字段用于添加 Grafana 自定义配置。在集群进行 deploy/scale-out/scale-in/reload 操作时，TiUP 会将 config 字段的内容会添加到 Grafana 的配置文件 grafana.ini 中
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:608]:
```suggestion
- `config`：该字段用于添加 Grafana 自定义配置。在集群进行部署、扩容、缩容、重新加载 (Reload) 等操作时，TiUP 会将 `config` 字段的内容会添加到 Grafana 的配置文件 `grafana.ini` 中。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -642,6 +646,7 @@ grafana_servers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `listen_host`：用于指定监听地址，使得 Alertmanager 可以通过代理访问。 推荐配置为 0.0.0.0。
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:649]:
```suggestion
- `listen_host`：用于指定监听地址，从而可以通过代理访问 Alertmanager。推荐使用 `0.0.0.0`。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 3]
```diff
@@ -554,6 +556,7 @@ tispark_workers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `additional_scrape_conf`： 自定义 Prometheus scrape 配置。更多信息，请参考[自定义监控组件的配置](/tiup/customized-montior-in-tiup-environment.md#自定义-prometheus-scrape-配置)
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:559]:
```suggestion
- `additional_scrape_conf`：自定义 Prometheus scrape 配置。在集群进行 deploy/scale-out/scale-in/reload 操作时，TiUP 会将 additional_scrape_conf 字段的内容会添加到 Prometheus 配置文件的对应参数中。更多信息，请参考[自定义监控组件的配置](/tiup/customized-montior-in-tiup-environment.md#自定义-prometheus-scrape-配置)
```

**Review by qqqdan — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 4]
```diff
@@ -554,6 +556,7 @@ tispark_workers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `additional_scrape_conf`：自定义 Prometheus scrape 配置。在集群进行 deploy/scale-out/scale-in/reload 操作时，TiUP 会将 additional_scrape_conf 字段的内容会添加到 Prometheus 配置文件的对应参数中。更多信息，请参考[自定义监控组件的配置](/tiup/customized-montior-in-tiup-environment.md#自定义-prometheus-scrape-配置)
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:559]: 相关内容来自这里 https://docs.pingcap.com/zh/tidb/stable/customized-montior-in-tiup-environment#%E8%87%AA%E5%AE%9A%E4%B9%89-prometheus-scrape-%E9%85%8D%E7%BD%AE

File: tiup/tiup-cluster-topology-reference.md [hunk 5]
```diff
@@ -602,6 +605,7 @@ monitoring_servers:
 - `password`：Grafana 对应的密码
 - `dashboard_dir`：该字段指定一个本地目录，该目录中应当含有完整的 `dashboard(*.json)` 文件，这些文件会在集群配置初始化阶段被传输到目标机器上，作为 Grafana 的 dashboards
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `config`：该字段用于添加 Grafana 自定义配置。在集群进行 deploy/scale-out/scale-in/reload 等操作时，TiUP 会将 `config` 字段的内容添加到 Grafana 的配置文件 `grafana.ini` 中。更多信息，请参考[自定义 Grafana 其他配置](/tiup/customized-montior-in-tiup-environment.md#自定义-grafana-其他配置)
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:608]: 相关内容来自 https://docs.pingcap.com/zh/tidb/stable/customized-montior-in-tiup-environment#%E8%87%AA%E5%AE%9A%E4%B9%89-grafana-%E5%85%B6%E4%BB%96%E9%85%8D%E7%BD%AE

File: tiup/tiup-cluster-topology-reference.md [hunk 6]
```diff
@@ -642,6 +646,7 @@ grafana_servers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `listen_host`：用于指定监听地址，从而可以通过代理访问 Alertmanager。推荐使用 `0.0.0.0`。更多信息，请参考[自定义 Alertmanager 配置](/tiup/customized-montior-in-tiup-environment.md#自定义-alertmanager-配置)
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:649]: 相关内容来自 https://docs.pingcap.com/zh/tidb/stable/customized-montior-in-tiup-environment#%E8%87%AA%E5%AE%9A%E4%B9%89-alertmanager-%E9%85%8D%E7%BD%AE

File: tiup/tiup-cluster-topology-reference.md (hunk 4 above)
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:559]:
```suggestion
- `additional_scrape_conf`：自定义 Prometheus scrape 配置。在集群进行 deploy/scale-out/scale-in/reload 操作时，TiUP 会将 `additional_scrape_conf` 字段的内容添加到 Prometheus 配置文件的对应参数中。更多信息，请参考[自定义监控组件的配置](/tiup/customized-montior-in-tiup-environment.md#自定义-prometheus-scrape-配置)
```

File: tiup/tiup-cluster-topology-reference.md (hunk 6 above)
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:649]:
```suggestion
- `listen_host`：指定监听地址，从而可以通过代理访问 Alertmanager。推荐使用 `0.0.0.0`。更多信息，请参考[自定义 Alertmanager 配置](/tiup/customized-montior-in-tiup-environment.md#自定义-alertmanager-配置)
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**